coucharchive

```diff
@@ -755,6 +818,9 @@ def main():
                         datefmt="%Y-%m-%dT%H:%M:%S%z",
                         level=loglevel)
 
+    has_run_queue_control = \
+        args.action != 'load' and not args.no_run_queue_control
```

Comment:
Hum OK why not!

Reply:
Without that the script crash.
Is it OK for you? Or you see a better solution?

Reply:
No, it's OK!